### PR TITLE
Update import paths

### PR DESCRIPTION
**Pull request by davelange**
Why:
- To avoid using complicated relative path imports

How:
- Update tsconfig paths entry with `src/` alias.
- Add `vite-tsconfig-paths` to make this work with vite.
- Update all import paths inside `renderer`

### CODE REVIEW

**Review by juliangruber — APPROVED** [review]
Nice!!

File: renderer/src/pages/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { useState } from 'react'
-import HeaderBackgroundImage from '../assets/img/header.png'
-import Modal from '../components/Modal'
-import ActivityLog from '../components/ActivityLog'
-import UpdateBanner from '../components/UpdateBanner'
-import WalletWidget from '../components/WalletWidget'
-import useStationActivity from '../hooks/StationActivity'
-import { formatTokenValue } from '../utils/number-ops'
+import HeaderBackgroundImage from 'src/assets/img/header.png'
```
Comment (bajtos) [inline renderer/src/pages/dashboard/Dashboard.tsx:2]: I am not a fan of letting TypeScript rewrite the import paths. When I look at this import, I need to recall that `src` is mapped in TypeScript config before I can figure out where to find the imported file. I find relative import path easier to reason about.

I guess this is not a big deal as long as IDEs like VS Code understand this TypeScript feature and can navigate around the project when using commands like "go to definition".

I am not actively working on the front-end, so you should not put too much weight to my opinions. Choose what works best for you.

Comment (juliangruber) [reply renderer/src/pages/dashboard/Dashboard.tsx:2]: I wasn't a fan of this at first either (when I first encountered it), but it's a very common convention in FE land and once you get used to it (especially like this where it's very simple) it makes deeper nesting much easier to work with

Comment (davelange) [reply renderer/src/pages/dashboard/Dashboard.tsx:2]: It makes nested files easier to work with, and as long as the rewritten path matches the actual file structure inside "src", I think it remains clear enough.

**Comment by davelange** [comment]
It seems I can't change this PRs base back to `filecoin-station:v2`, so I'll create a new PR and reference this there.

**Pull request closed without merging**